documentation/Overview.md

```diff
@@ -1,28 +1,32 @@
 
 This folder contains instructions for developing SDK plugins and utilizing the SDK's various features.
 
+# Glossary
+
+For quick definitions/examples of terms and concepts, refer to the [Glossary](./Glossary.md). 
+
 # Requirements
 The following are required in order to develop an SDK plugin:
 * [NuGet](https://www.nuget.org/downloads)
 * [.NET Standard 2.0](https://dotnet.microsoft.com/download/visual-studio-sdks)
 * A text editor (for editing your source code)
 
-It is recommended to use [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) to develop an SDK plugin since it satisfies all three requirements. Future documentation assumes the use of Visual Studio, but the instructions may be adapted for other editors/IDEs.
+It is recommended to use [Visual Studio](https://visualstudio.microsoft.com/downloads/) to develop an SDK plugin since it satisfies all three requirements. Future documentation assumes the use of Visual Studio, but the instructions may be adapted for other editors/IDEs.
 
 # Getting the SDK
-The SDK is currently published as a NuGet package under the name [Microsoft.Performance.SDK](https://www.nuget.org/packages/Microsoft.Performance.SDK/). 
+The SDK is published as a NuGet package under the name [Microsoft.Performance.SDK](https://www.nuget.org/packages/Microsoft.Performance.SDK/). 
 Since it is hosted on [NuGet.org](https://www.nuget.org/), it can be added to a `csproj` with no additional configuration by using 
 the Visual Studio NuGet Package Manager, `dotnet.exe`, or `nuget.exe`.
 
-# Creating Your First Project
-Please see the [Creating your first project](Using-the-SDK/Creating-your-project.md)
+# Creating Your First Plugin
+Refer to [Creating your first plugin](Using-the-SDK/Creating-your-plugin.md)
 
 # Recommended Reading Order
 To best understand how the SDK works and how to develop SDK plugins, it is recommended to read documentation in the following order:
-1) [Architecture/Overview](./Architecture/Overview.md) to understand at a high level the various system the SDK provides
+1) [Architecture/Overview](./Architecture/Overview.md) to understand at a high level the various systems the SDK provides
 2) [Architecture/The Data Processing Pipeline](./Architecture/The-Data-Processing-Pipeline.md) to understand how to systematically process data that 
 can be used by tables
 3) [Using the SDK/Creating an SDK Plugin C# Project](Using-the-SDK/Creating-your-project.md) to get your developer environment ready to create an SDK plugin
```

Comment:
we could add a 4th line here for referencing the glossary

Reply:
It's already in this doc. I also don't think it's a "recommended reading," it's more of a reference.